core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -2062,6 +2062,11 @@ public static String getSyncMetricName(long mountId) {
               + "Use this metric to monitor the RPC pressure on worker.")
           .setMetricType(MetricType.GAUGE)
           .build();
+  public static final MetricKey WORKER_MASTER_REGISTRATION_SUCCESS_COUNT =
```

Comment:
the former one

Reply:
what does this count mean? Like if you see the count is 100, it could mean the worker retried connecting to one particular master, or the worker restarted a few times and connected to all masters. I just feel we could probably discuss if there are certain events you'd like to capture, then we update the metric accordingly?

Reply:
yeah but you can tell how many registration happen in a given time window which gives you some insights. e.g. if the cluster have been running for a while and there's tons of registration events, it might indicate something funky happens

Reply:
if you want we can split this metric to registration count plus re-registration count if you see that will be more helpful

Reply:
okay so we discussed offline and the conclusion is that we merge this metrics for now and we can iterate if users need more visibility or clarity on this